### PR TITLE
Remove upload blob from detector init. Blob upload should not be nece…

### PR DESCRIPTION
**Pull request by kaiaw**
…ssary as it is included in the hpk

### CODE REVIEW

**Comment by kaiaw** [comment]
@jontore @brikendr Let's discuss if we should do this tomorrow

File: src/components/detector.ts [hunk 1]
```diff
@@ -4,7 +4,6 @@ import IDeviceManager from './../interfaces/iDeviceManager';
 import iDetectorOpts, { DetectionConvertion } from './../interfaces/IDetectorOpts';
 import Api from './api';
 import CameraEvents from './../utilitis/events';
-import axios from 'axios';
 import { getFramingConfig } from './../utilitis/framingconfig';
```
Comment (brikendr) [inline src/components/detector.ts]: Could you also remove `_defaultBlobURL` and `_defaultConfigURL` class properties?

**Comment by brikendr** [comment]
@kaiaw i think if we go with this change, than any cameras that have an old version that do not have a persistent blob wont work i suppose!

**Comment by kaiaw** [comment]
Yes, I suppose that is fine

**Pull request merged**